README.md

```diff
@@ -132,81 +129,66 @@ We currently support the matrix below.
+- First pass on localization complete.  17 different languages.  We know there will be some rough areas, please [make us aware so we can correct them][loc-bug].
+- Logging added into the installer
+- Large sums of accessibility issues fixed.
+- Less notifications for installing
+- FxCop work is almost fully wrapped up
 
-- Localization pipeline is flowing from our Github to the loc system and back.  0.25 should be localized now.
-- The EXE installer should be at parity now with the MSI.  Please go to the wiki for [installer args](https://github.com/microsoft/PowerToys/wiki/Installer-arguments-for-exe)
+**Color Picker**
+- Additional color style selections such as 
 
 **FancyZones**
-
-- Fixed bug on not seeing a newly attached screen
-- Fixed spanning across monitors bug
-- Added in default layout for new users, a Priority Grid
-- Added keyboard support to grow / shrink to multiple zones
-- General bug fixes
-
-**PT Run**
-
-- Multiple crash bugs fixed.  Prioritized any users reported along with top hits from Watson reporting
-- Stopped PT Run from interfering with an install
-- Fixed folder bug if it had a # in it (Thanks @jjw24 for the PR!)
-- Fixed a screen flicker for 
-- General bug fixes
-- Allow Command Line args in PowerToys Run (Thanks @@royvou)
+- Multiple bugs fixed
+- Better zone drawing improvements
 
 **Keyboard manager**
-
-- Multiple crash bugs fixed.  Prioritized any users reported along with top hits from Watson reporting
-- Fixed multiple accessibility issues.
-- General bug fixes
-
-**Preview Pane**
-
-- Added in Frontmatter and better (but still basic) latex support. 
-
-**Settings**
-
-- Fixed scaling issue for responsive design on Image Resizer
-- Fixed crash on empty color value.
-- Fixed crash for toggling FancyZones on/off
-- Fixed 0x00 NFTS crash for settings
-- Fixed multiple accessibility issues.
-- Layout adjustments (Thanks @niels9001)
-- General bug fixes
-
-**Dev related**
-
-- FxCop is being rolled out across all PowerToys. This should catch a lot of possible leaks.
-- Unified PT Run's log system
-- PT Run's calc plugin now has unit tests (Thanks @P-Storm)
-- Dev setup install script now supports VS preview (Thanks @TobiasSekan)
-- @CaelestisZ, @kameshkotwani, @adriancampos, @RahulDas782 for doc tweaks
-- Thanks @Aaron-Junker, @jay-o-way and @htcfreek for helping triage!
-- Thanks for everyone that filled an issue.  It really does help us prioritize
+- Fixed terminal input map failure
+- Better app compat
+- Multiple bug fixes
+- Ability to directly disable keys/shortcuts
+
+**PowerToys Run**
+- expanded environment var searching such as %windr%
+- multiple crash bug fixes
+- Improvements on calculator plugin
+- Directly able to override theming
+- Windows will open to what shell you want
+- Better action key support
+  - `=` for direct calculator
+  - `?` for direct file searching
+  - `.` for direct for applications
+  - `//` for direct URL
+  - `<` for running processes
+  - `>` for shell processes
```

Comment:
Additional:
- Support to search app by exe name
- Fix for apps can't start with parameters as admin

BTW: We should document how an user can start apps with params. We have missed to mention this feature in the 0.23 release notes. (See #5791)